src/app/calculator/furnaces/flue-gas/flue-gas-results/flue-gas-results.component.html

```diff
@@ -0,0 +1,105 @@
+<label class="group-label">Baseline Results</label>
+<table #copyTable0 class="table table-hover table-striped">
+  <tbody>
+    <tr>
+        <td class="bold w-50">Fuel Use</td>
+        <td *ngIf="output.baseline.fuelUse" class="text-center w-50">
+          {{output.baseline.fuelUse | number:'1.0-0'}}
+          <span class="input-group-addon">
+            <span [innerHTML]="output.baseline.energyUnit | settingsLabel"></span></span>/yr
+        </td>
+        <td *ngIf="!output.baseline.fuelUse" class="text-center w-50">
+          &mdash; &mdash;
+        </td>
+      </tr>
+      <tr>
+        <td class="bold w-50">Fuel Cost</td>
+        <td *ngIf="output.baseline.fuelCost" class="text-center w-50">
+          {{output.baseline.fuelCost | currency:'USD':'symbol':'1.2-2'}}
+        </td>
+        <td *ngIf="!output.baseline.fuelCost" class="text-center w-50">
+          &mdash; &mdash;
+        </td>
+      </tr>
+      <tr>
+        <td class="bold w-50">Gross Loss</td>
+        <td *ngIf="output.baseline.flueGasLosses" class="text-center w-50">
+          {{output.baseline.flueGasLosses | sigFigs:'3'}}
+          <span class="input-group-addon">
+            <span [innerHTML]="output.baseline.energyUnit | settingsLabel"></span></span>/hr
+        </td>
+        <td *ngIf="!output.baseline.flueGasLosses" class="text-center w-50">
+          &mdash; &mdash;
+        </td>
+      </tr>
+  </tbody>
+</table>
+<!-- export table component -->
+<app-exportable-results-table [tableString]="table0String" (updateTableString)="updateTable0String()">
+</app-exportable-results-table>
```

Comment:
For the results, you can just have one of the export buttons. You can add together the table strings from the different tables and pass them into the component.